python/django_bolt/workers.py

```
@@ -0,0 +1,272 @@
+    (macOS/BSD). Returns None for dead PIDs or unsupported platforms.
+    """
+    try:
+        with open(f"/proc/{pid}/status", "rb") as status_file:
+            for line in status_file:
+                if line.startswith(b"VmRSS:"):
+                    return int(line.split()[1]) * 1024  # VmRSS is in KiB
+    except (OSError, ValueError, IndexError):
+        pass
+
+    try:
+        result = subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)